_pkgdown.yml

```diff
@@ -10,6 +10,9 @@ template:
   #   border-radius: 0.5rem
   #   btn-border-radius: 0.25rem
 
+development:
+  mode: devel
+
```

Comment:
added this, so that next time the websuite will bu built, the result will be stored under docs/dev.
To build the release version (for the next release), mode must be temporarily switched to **release**